FeedStoreChallenge/CoreData/Entities/ManagedFeedImage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Антон on 5.09.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
```

Comment:
`cache` property is marked as optional in `FeedStore.xcdatamodel`, please make it optional to make the model consistent.
```suggestion
	@NSManaged var cache: ManagedCache?
```